scripts/release-binary.sh

```diff
@@ -211,6 +211,14 @@ trap 'rm -rf ${TMP_WASM}' EXIT
 make build_wasm
 if [ -n "${DST}" ]; then
   for extension in "${extensions[@]}"; do
+    if [ "${PUSH_DOCKER_IMAGE}" -eq 1 ]; then
```

Comment:
can we use this image when `telemetry.metadataExchange.wasmEnabled: true`

Reply:
BTW, do we have any data about the performance between `wasm metadataExchange` and `null metadataExchange`?

Reply:
We used to measure it a long time ago. It is roughly 10% ~ 20% more expensive both cpu and latency. Although I heard it has been improved, so a new measurement probably would be needed.

Reply:
once https://perf.dashboard.istio.io/ is fixed, we can setup a measurement daily.